pom.xml

```diff
@@ -68,13 +66,6 @@
                 <artifactId>commons-codec</artifactId>
                 <version>1.16.1</version>
             </dependency>
```

Comment:
I (Informational): This round of upgrades allowed this workaround to be removed with the `5.8.1` upgrade of `net.masterthought:cucumber-reporting`.